openpower_guard_interface.cpp

```diff
@@ -87,5 +87,40 @@ void clear(const uint32_t recordId)
     }
 }
 
+GuardRecords getAll()
+{
+    GuardRecords records;
+    try
+    {
+        records = libguard::getAll();
+    }
+    catch (libguard::exception::GuardFileOpenFailed& e)
```

Comment:
Looks like same exception /trow in another commit , , keep inline function handle catch?